### PR TITLE
Add more Korean translations and corrections in high standard

### PR DESCRIPTION
**Pull request by ianpark**
## Summary:
I have been driving Korean translation of Condenser website. Korean Steemians have been vigorously contributed to this project, and it proves the significancy of supporting proper Korean language. Many thanks to the volunteers, we have reached near professional translation standard. Please take a look and help this project continue until it accomplishes the perfection.

## Contributed Steemians:
@bree1042 @chocolate1st @sjchoi @feyee95 @crowsaint @happyworkingmom @clayop @coffeenut @kmlee @sinner264 @seaturtle @yangmok701 and me @asbear.

## The related postings:
https://steemit.com/kr/@chocolate1st/4xqwlh-or-or-or-or-1
https://steemkr.com/kr-translation/@bree1042/steemit-translation-project
https://steemit.com/kr-translation/@bree1042/2-translate-steemit-into-korean
https://steemit.com/kr/@asbear/74wrbd
https://steemit.com/kr/@asbear/2etp95-steemit-com-korean-translation-project
https://steemit.com/kr/@asbear/steemit-com-korean-translation-project
https://steemit.com/kr/@sjchoi/5ls9us

This project is driven volunteers. Anyone thinks this project helps Steemit's user growth, please vote on these postings as well as following in the future.

Thanks!

### CODE REVIEW

**Comment by relativityboy** [comment]
@gl2748 - very nice catch. Checking now. @ianpark likely, because the root of Steem is _esteem_ and is a proper-name we'll want to keep it Alpha.

File: src/app/locales/ko.json [hunk 1]
```diff
@@ -652,17 +652,17 @@
...
-            "This operation requires your %(authType)s key or Master password.",
-        "keep_me_logged_in": "Keep me logged in",
+            "이 작업은 %(authType)s 키 또는 마스터 비밀번호가 필요합니다.",
+        "keep_me_logged_in": "로그인 유지",
         "amazing_community": "amazing community",
         "to_comment_and_reward_others": " to comment and reward others.",
         "signup_button": "Sign up now to earn ",
         "signup_button_emphasis": "FREE STEEM!",
-        "sign_up_get_steem": "Sign up. Get STEEM",
+        "sign_up_get_steem": "계정을 만들고 스팀을 채굴하세요.",
```
Comment (relativityboy) [inline src/app/locales/ko.json]: Hold for the correct translation of Steem.

Comment (relativityboy) [reply src/app/locales/ko.json]: @ianpark please change this line to read something like `계정을 만들고 광산 STEEM.`

Comment (ianpark) [reply src/app/locales/ko.json]: ok good point, as it's currency not the brand. thanks!

**Comment by ianpark** [comment]
@gl2748 @relativityboy
Thanks for your review. Korean most of time do not use english as is in professional web site. Chocopie is 초코파이 on the product package in Korea. We prefer using Korean on brand names all the time unless it needs to be English. Google translator may translate 스팀 to Steam as Steam is more like a proper word, but it's just translator's problem, soon after Steem is massively being used in Korea, 스팀 will be translated to Steem I believe. Using "Steem" as is in English on Steemit website would look unprofessional. The korean people I work with for translation are all working professionals working on IT, design, literature etc and we confirm that 스팀 fits much better.
Still I made one thing to "STEEM" as it is the name of currency. STEEM DOLLAR is called 스팀달러, and trust us, it's significantly better. STEEM POWER is also translated to 스팀달러, which is way better than using the word as is. Hope I answered to your questions :)

**Comment by originated** [comment]
@ianpark Thanks for your input it's greatly appreciated. Please have any community members join our unified translation platform by emailing translate@steemit.com - we can combine the efforts

**Comment by ianpark** [comment]
@originated 
Thanks for the suggestion. It could be great, I will announce it to the translation volunteers.
You may know that Korean translation needs multiple steps for achieving professional level result due to the the gap between the two languages. Therefore individual contribution may not end up with the best result automatically. That's why I orchestrate the project with continuous discussion and review.

Btw, I applied the change, so please anyone continue reviewing it :)

**Comment by ianpark** [comment]
@originated Hi, thanks for delivering this change. Now the Steemit in Korean setting looks much better and we are slightly amused by that ;)
Btw can you explain more about the translation platform? What do the individual kr translators need to send to translate@steemit.com for joining there?

**Pull request merged**